src/Microsoft.DotNet.XHarness.CLI/Commands/WASM/JS/WasmTestCommand.cs

```diff
@@ -56,12 +56,16 @@ protected override async Task<ExitCode> InvokeInternal(ILogger logger)
             var engineBinary = Arguments.Engine.Value switch
             {
                 JavaScriptEngine.V8 => "v8",
+                JavaScriptEngine.NodeJS => "node",
                 JavaScriptEngine.JavaScriptCore => "jsc",
                 JavaScriptEngine.SpiderMonkey => "sm",
                 _ => throw new ArgumentException("Engine not set")
             };
 
-            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            if (engineBinary.Equals("node"))
```

Comment:
Why not just return `node.exe` from the switch?

Reply:
Let's test how it ships on linux too

Reply:
Agreed, I actually had the same idea after I pushed but it took a while before I could actually test it haha. I just tested it on linux and windows and seems like it works as expected now